Sources/KlaviyoUI/InAppForms/IAFWebViewModel.swift

```diff
@@ -41,6 +41,11 @@ class IAFWebViewModel: KlaviyoWebViewModeling {
         let sdkVersionScript = "document.head.setAttribute('data-sdk-version', '\(sdkVersion)');"
         let sdkVersionWKScript = WKUserScript(source: sdkVersionScript, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
         loadScripts?.insert(sdkVersionWKScript)
+
```

Comment:
nit suggestion: can we make this a little modular like, 
```swift
func initializeLoadScripts() {
      loadScripts?.insert(sdkNameWKScript)
      loadScripts?.insert(sdkVersionWKScript)
      loadScripts?.insert(handshakeWKScript)
}
```
and each script is a computed var or a method to make reading this a little easier? Not a blocker by any means.

Reply:
I will accept the compliment for Belle's work!